src/containers/CareerDetails.js

```diff
@@ -1,19 +1,123 @@
 import React, { Component } from 'react';
-import styled from 'styled-components';
+import { connect } from 'react-redux';
+import {
+  Icon,
+  CareerLogo,
+  TitleWrapper,
+  Description,
+  TopSection,
+  CareerTagline,
+  TitleTagWrapper,
+  CareerContainer,
+  CareerTitle,
+  SectionTitle,
+  DetailSection,
+  SalaryStart,
+  Start,
+  SalaryTenYear,
+  TenYear,
+  TableContainer,
+  TableColumn,
+  GradeBagrut,
+  ReqBagrut,
+  UserBagrut,
+  GradePsyc,
+  ReqPsyc,
+  UserPsyc,
+  GradeTawjihi,
+  ReqTawjihi,
+  UserTawjihi,
+} from './../styles/details.js';
 
-const CareerContainer = styled.div`
-  background-color: red;
-`;
-const MainTitle = styled.h1``;
+import { fetchDetails } from '../actions/career_details';
 
 class CareerDetails extends Component {
+  componentDidMount() {
+    this.props.fetchDetails({ career_id: this.props.match.params.id });
+  }
+
+  showDetails = () => {
+    if (this.props.careerDetail.dataFetched) {
+      return (
+        <div>
+          <TopSection>
+            <TitleTagWrapper>
+              <CareerTitle>
+                {this.props.careerDetail.career.career[0].title_ar}
+              </CareerTitle>
+              <CareerTagline>
+                {this.props.careerDetail.career.career[0].tagline_ar}
+              </CareerTagline>
+            </TitleTagWrapper>
+            <CareerLogo className="material-icons">
+              {this.props.careerDetail.career.career[0].icon_name}
+            </CareerLogo>
+          </TopSection>
+
+          <DetailSection>
+            <Description>
+              {this.props.careerDetail.career.career[0].description_ar}
+            </Description>
+            <TitleWrapper>
+              <SectionTitle>Salary</SectionTitle>
+            </TitleWrapper>
+            <TableContainer>
+              <TableColumn>
+                <Start>
+                  ₪ {this.props.careerDetail.career.career[0].salary_start}
+                </Start>
+                <TenYear>
+                  ₪ {this.props.careerDetail.career.career[0].salary_ten_year}
+                </TenYear>
+              </TableColumn>
+              <TableColumn>
+                <SalaryStart>Junior </SalaryStart>
+                <SalaryTenYear>Senior</SalaryTenYear>
+              </TableColumn>
+            </TableContainer>
+          </DetailSection>
+
+          <DetailSection>
+            <TitleWrapper>
+              <Icon className="material-icons">keyboard_arrow_left</Icon>
+              <SectionTitle>Haifa University</SectionTitle>
+              <Icon className="material-icons">keyboard_arrow_right</Icon>
+            </TitleWrapper>
+            <TableContainer>
+              <TableColumn>
+                <ReqBagrut> 100.00 </ReqBagrut>
```

Comment:
I think all the available universities should be listed here without hard coded grades and then in a later PR we can add the carousel functionality.

Reply:
Maybe there should also be an indication of  which column is university requirement and which is the user.